### PR TITLE
Add parallel-put to same object.

### PR DESCRIPTION
**Pull request by harshavardhana**
This test tests the sanctity of distributed locking and cleanup
when servers are being repeatedly taken down and brought
backup in succession.

### CODE REVIEW

**Comment by harshavardhana** [comment]
ping @hackintoshrao

**Comment by hackintoshrao** [comment]
Yes @harshavardhana .

**Comment by harshavardhana** [comment]
> Yes @harshavardhana .

Not yes - review :-)

File: parallel-put-lock/parallel-put.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+	// Data to be uploaded.
+	data := make([]string, len(clnts))
+	for i := range data {
+		data[i] = strings.Repeat(fmt.Sprintf("Hello, World - %d", i), 10)
+	}
+
+	// Continously write to all nodes.
+	j := 0
+	for {
```
Comment (hackintoshrao) [inline parallel-put-lock/parallel-put.go:66]: When does the test stop ? Does it have to be manually stopped.

Comment (harshavardhana) [reply parallel-put-lock/parallel-put.go:66]: CTRL-C :+1:

Comment (harshavardhana) [reply parallel-put-lock/parallel-put.go:66]: This is also meant for locking and unlocking on the same file across many nodes in parallel.

**Comment by hackintoshrao** [comment]
Handy test, can be used for various tests. LGTM.

**Review by fwessels — COMMENTED** [review]
Nicely done test, so simple to setup. Some performance measures may be handy to be logged, eg objects uploaded per second. 

And can we expand it for an non-localhost setup? (Thus actually across multiple servers?)

**Comment by harshavardhana** [comment]
Yes that would work too.

**Pull request merged**